examples/kafka/envoy.yaml

```diff
@@ -0,0 +1,38 @@
+static_resources:
+  listeners:
+  - address:
+      socket_address:
+        address: 0.0.0.0 # Host that Kafka clients should connect to.
+        port_value: 19092  # Port that Kafka clients should connect to.
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.kafka_broker
+        typed_config:
+          "@type": type.googleapis.com/envoy.config.filter.network.kafka_broker.v2alpha1.KafkaBroker
```

Comment:
I know this is jumping a bit outside of this PR for the sandbox but the actual Kafka documentation does not use the proper `typed_config` which it should. RST to be updated at https://github.com/envoyproxy/envoy/blob/master/docs/root/configuration/listeners/network_filters/kafka_broker_filter.rst. I can open a separate PR or ask @adamkotwasinski to add to his as part of his Kafka version update PR at https://github.com/envoyproxy/envoy/pull/9582

Reply:
👍 Thanks! I'll fix it there